docs/changelog.md

```diff
@@ -50,6 +50,12 @@ permalink: /changelog/
 * **concept-menu**
   * 🌟 Added `AsyncDrawableMenuIcon` class to use icons in a menu that will be loaded later.
 
+* **service-glean**
```

Comment:
I think you need to add this in the 62 section at the beginning of the doc

Reply:
I should get more coffee first.